README.md

```diff
@@ -351,19 +351,24 @@ Compensation comes in the forms of cash pay (salary and any bonuses) and any non
 including [benefits](https://en.wikipedia.org/wiki/Employee_benefits#United_States) like
 health insurance, family-related protections, perks, and retirement plans.
 
-🄳 For jobs in many startups and established companies alike, compensation includes some
-form of ownership or likely future ownership in the company;
-this ownership is called **equity**.
-
-🄳 [**Equity compensation**](http://www.investopedia.com/terms/e/equity-compensation.asp) is the
-practice of granting equity in exchange for work.
+Company strategies for compensation are
+[far from simple](http://firstround.com/review/counterintuitive-comp-tips-for-the-unwary-and-uninitiated/).
+Beth Scheer of venture fund Homebrew offers
+[a thoughtful overview](https://quip.com/HEB3Ah9dYD6o) of compensation in startups.
```

Comment:
"Beth Scheer, head of talent at the venture fund Homebrew," (or just, "Beth Scheer of _the_ venture fund Homebrew") 
"a [thoughtful overview]"